serializers/err.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const _ = require('lodash');
+const bunyan = require('bunyan');
+
+function err(err) {
```

Comment:
Could you add some jdoc please? Also maybe rename the `err` parameter to `error` so it's not confused with the name of the function?